fuels-abigen-macro/tests/test_projects/token-ops/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+authors = ["Fuel Labs"]
+edition = "2021"
+license = "Apache-2.0"
+name = "token-ops"
+version = "0.1.0"
```

Comment:
Use 0.0.0 for things that aren't meant to be published.

Reply:
We actually don't need any of these `Cargo.toml`s because we're just using the compiled contracts and ABI for testing.

Reply:
Ah, then maybe remove altogether?

Reply:
Yup!